bayesian/utility/v2/ca_response_builder.py

```diff
@@ -128,17 +128,27 @@ def get_message(self):
             message += f"{self.public_vul} known security vulnerability "
             message += f"and {self.pvt_vul} security advisory with {len(self.severity)} " \
                        f"having {self.severity[0]} severity. "
+
+            if not self.nocve_version:
+                message += 'Currently we don\'t have any recommended version.'
```

Comment:
Was there a message earlier?
If you send this message do you know what the UI will display @dvandra

Reply:
> Was there a message earlier?

@prashbnair  Yes, Please refer to Jira ticket attached.

Reply:
Can you please change the message to "No recommended version"

Reply:
Ack!